import.go

```diff
@@ -26,13 +25,13 @@ func importBuilder(store storage.Store, options ImportOptions) (*Builder, error)
 	systemContext := getSystemContext(options.SignaturePolicyPath)
 
 	if c.ImageID != "" {
-		ref, err := is.Transport.ParseStoreReference(store, "@"+c.ImageID)
-		if err != nil {
-			return nil, fmt.Errorf("no such image %q: %v", "@"+c.ImageID, err)
+		ref, err2 := is.Transport.ParseStoreReference(store, "@"+c.ImageID)
+		if err2 != nil {
+			return nil, fmt.Errorf("no such image %q: %v", "@"+c.ImageID, err2)
 		}
-		src, err := ref.NewImage(systemContext)
-		if err != nil {
-			return nil, fmt.Errorf("error instantiating image: %v", err)
+		src, err3 := ref.NewImage(systemContext)
+		if err3 != nil {
+			return nil, fmt.Errorf("error instantiating image: %v", err3)
 		}
 		defer src.Close()
 		config, err = src.ConfigBlob()
```

Comment:
Not an issue, but I'm not getting why you added numbers to some but not all 'err' variables.